DiamondLightSource/SynchWeb
# LIMS-1436: Use auth provider provided logout URL when SSO is enabled

**JIRA ticket**: [LIMS-1436](https://jira.diamond.ac.uk/browse/LIMS-1436)

**Summary**:

In order to make the authentication controllers truly generic, the logout URL must be provided by the upstream SSO authentication provider, to which logout attempts must be redirected to.

**Changes**:
- Remove logout page
- Redirect user to SSO logout if SSO is enabled

**To test**:
- Set `cas_sso` in `config.php` to `true`, set `authentication_type` to `oidc`, Log in, click the log out button, ensure you're redirected to CAS
- Set `cas_sso` in `config.php` to `true`, set `authentication_type` to `combined`, Log in, click the log out button, ensure you're redirected to CAS
- Set `cas_sso` in `config.php` to `false`, set `authentication_type` to `dummy`, ensure you're not redirected to CAS when logging out
